src/components/ClientSideLoggingToolMenu/BaseClientSideLoggingToolMenu.tsx

```diff
@@ -30,9 +32,22 @@ type BaseClientSideLoggingToolProps = {
     onDisableLogging: (logs: Log[]) => void;
     /** Action to run when enabling logging */
     onEnableLogging?: () => void;
+    /** Boolean to know if this was opened via test tools modal */
+    isViaTestToolsModal: boolean;
+    /** Action to close the test tools modal */
+    closeTestToolsModal?: () => void;
 } & BaseClientSideLoggingToolMenuOnyxProps;
 
-function BaseClientSideLoggingToolMenu({shouldStoreLogs, capturedLogs, file, onShareLogs, onDisableLogging, onEnableLogging}: BaseClientSideLoggingToolProps) {
+function BaseClientSideLoggingToolMenu({
+    shouldStoreLogs,
+    capturedLogs,
+    file,
+    onShareLogs,
+    onDisableLogging,
+    onEnableLogging,
+    isViaTestToolsModal,
+    closeTestToolsModal,
```

Comment:
```suggestion
    closeTestToolsModal = () => {},
```

Then we won't need `if (closeTestToolsModal)` check on line 93